frontend/apps/app/package.json

```
@@ -41,6 +43,7 @@
     "lint:biome": "biome check .",
     "postinstall": "cp ../../packages/db-structure/node_modules/@ruby/prism/src/prism.wasm prism.wasm",
     "start": "next start",
+    "trigger:dev": "pnpm dlx trigger.dev@latest dev",
```

I would like to add to devDependencies, and fix to the following:

```suggestion
    "trigger:dev": "trigger.dev dev",
```

---

Thanks for letting me know!
I added `trigger.dev`  to devDependencies. (https://github.com/liam-hq/liam/pull/846/commits/21b1d8b5367faad377d946589dd6f063267df48b)

But Dependency Review test faild because trigger.dev depends on `esbuild@0.23.1`
So, I added `resolutions` to `package.json` because `trigger.dev` depends on `esbuild@0.23.1`.
Is that OK?🙏

https://github.com/liam-hq/liam/pull/846/commits/21b1d8b5367faad377d946589dd6f063267df48b ... add resolutions
https://github.com/liam-hq/liam/pull/846/commits/89e715ecc0df962f684676e55c16406045efd81f ... change packageManager

---

Thank you for confirming!😄